modules/administration-guide/pages/running-at-scale.adoc

```diff
@@ -0,0 +1,206 @@
+
+This is particularly important for large clusters where the number of namespaces and operators can quickly add up to a significant amount of data. Disabling Copied CSVs can help reduce the load on etcd, leading to improved performance and responsiveness of the cluster.
+Additionally, it can help reduce the memory footprint of OLM, as it no longer needs to maintain and manage these additional resources.
+
+[NOTE]
+====
+You can find more details about "Disabling Copied CSVs" in the link:https://olm.operatorframework.io/docs/advanced-tasks/configuring-olm/#disabling-copied-csvs[official documentation].
+====
+
+.Cluster Autoscaling
+
+Although cluster autoscaling is a powerful {kubernetes} feature, you cannot always fall back on it. You should always consider predictive scaling by analyzing load data on your environment to detect daily or weekly usage patterns. If your workloads follow a pattern and there are dramatic peaks throughout the day, you should consider provisioning worker nodes accordingly. For example, if you have a predictable load pattern where the number of workspaces increases during business hours and decreases during off-hours, you can use predictive scaling to adjust the number of worker nodes based on the expected load.
+This can help ensure that you have enough resources available to handle the peak load while minimizing costs during off-peak hours.
+
+[NOTE]
+====
+Consider leveraging open-source solutions such as link:https://karpenter.sh/[Karpenter] for configuration and lifecycle management of the worker nodes. Karpenter can dynamically provision and optimize worker nodes based on the specific requirements of the workloads, helping to improve resource utilization and reduce costs.
+====
+
+.Multi-cluster
+
+By design, {prod} is not multi-cluster aware, and you can only have one instance per cluster.
+However,
+you can run {prod} in a multi-cluster environment by deploying {prod} in each different cluster
+and using a load balancer or DNS-based routing to direct traffic to the appropriate instance based on the user’s location or other criteria.
+This approach can help
+improve performance and reliability by distributing the workload across multiple clusters
+and providing redundancy in case of cluster failures.
+
+From the infrastructure perspective, the Developer Sandbox consists of multiple link:https://www.redhat.com/en/technologies/cloud-computing/openshift/aws[ROSA] clusters. On each cluster, the productized version of {prod} is installed and configured using link:https://argo-cd.readthedocs.io/en/stable/[Argo CD]. Since the user base is spread across multiple clusters, link:https://workspaces.openshift.com/[workspaces.openshift.com] is used as a single entry point to the productized {prod} instances. You can find implementation details about the multicluster redirector in the following link:https://github.com/codeready-toolchain/crw-multicluster-redirector[GitHub repository].
```

Comment:
@deerskindoll you should add more context about Developer Sandbox before introducing infrastructure